packages/main/cypress/specs/Calendar.cy.ts

```diff
@@ -0,0 +1,844 @@
+import { html } from "lit";
+import type Calendar from "../../src/Calendar.js";
+import type ToggleButton from "../../src/ToggleButton.js";
+import "../../src/ToggleButton.js";
+import "../../src/Calendar.js";
+import "../../src/Button.js";
+import "../../src/CalendarDate.js";
+import "@ui5/webcomponents-localization/dist/features/calendar/Islamic.js";
+import "@ui5/webcomponents-localization/dist/features/calendar/Gregorian.js";
+
+describe("Calendar general interaction", () => {
+	beforeEach(() => {
+		cy.mount(html`<ui5-calendar id="calendar1"></ui5-calendar>`);
+	});
+
+	it("Calendar is rendered", () => {
```

Comment:
Removing this test causes the below one to fail. There is some initialization timing issue, which I am currently investigating.